lib/src/authentication/widgets/input_password.dart

```diff
@@ -18,6 +19,10 @@ class InputPassword extends StatefulWidget {
 
   final TextInputAction textInputAction;
 
+  /// Controller of the main password input,
+  /// meaning that this input is used to confirm the first one.
+  final TextEditingController? passwordToConfirmController;
```

Comment:
I'm a little bit sceptical about this one 🤔 Instead we could have a `String? Function(String value) validate` method called. It will also allow us to push a specific password format validation. What do you think? 🤔

Reply:
It makes sense
Changed